pkg/cache/application.go

```diff
@@ -518,6 +518,10 @@ func (app *Application) onReserving() {
 // onReservationStateChange is called when there is an add or a release of a placeholder
 // If we have all the required placeholders progress the application status, otherwise nothing happens
 func (app *Application) onReservationStateChange() {
+	if app.originatingTask != nil {
+		events.GetRecorder().Eventf(app.originatingTask.GetTaskPod().DeepCopy(), nil, v1.EventTypeNormal, "Placeholder Allocated",
```

Comment:
For the action and reason, just use the text "GangScheduling", we already do it in Task.